### PR TITLE
CHE-1037: fix pulling of docker images

### PR DESCRIPTION
**Pull request by garagatyi**
@skabashnyuk @evoevodin Please review

### CODE REVIEW

File: plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/local/interceptor/EnableOfflineDockerMachineBuildInterceptor.java [hunk 1]
```diff
@@ -64,7 +64,7 @@ private void pullImage(String image, final LineConsumer creationLogsOutput)
         DockerImageIdentifier imageIdentifier = DockerImageIdentifierParser.parse(image);
         final ProgressLineFormatterImpl progressLineFormatter = new ProgressLineFormatterImpl();
         dockerConnector.pull(imageIdentifier.getRepository(),
-                             imageIdentifier.getTag(),
+                             imageIdentifier.getTag() == null ? "latest" : imageIdentifier.getTag(),
```
Comment (voievodin) [inline plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/local/interceptor/EnableOfflineDockerMachineBuildInterceptor.java]: `firstNonNull(imageIdentifier.getTag(), "latest)` looks better for me.

Comment (garagatyi) [inline plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/local/interceptor/EnableOfflineDockerMachineBuildInterceptor.java]: +1

Comment (garagatyi) [inline plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/local/interceptor/EnableOfflineDockerMachineBuildInterceptor.java]: done

**Comment by voievodin** [comment]
ok

**Pull request merged**